addons/docs/docs/recipes.md

```diff
@@ -58,7 +59,7 @@ basic.story = {
 
 ```md
 import { Meta, Story } from '@storybook/addon-docs/blocks';
-import * as stories from './Button.stories.js';
+import \* as stories from './Button.stories.js';
```

Comment:
Is this backlash on purpose?

Reply:
good catch. somehow VSCode/prettier inserts this automatically. will fix.